routes/medication/schema.js

```diff
@@ -0,0 +1,61 @@
+/* istanbul ignore file */
+const Schema = {
+  firstname: {
+    isLength: {
+      errorMessage: 'errors.firstname.length',
+      options: { min: 3, max: 200 },
```

Comment:
Had a validation error
![image](https://user-images.githubusercontent.com/950486/69441090-fdb30c00-0d17-11ea-9c81-66a1b310fb7a.png)

Reply:
Should be fixed now.